src/templates/audioPlayer.ejs

```diff
@@ -0,0 +1,8 @@
+<% if (zone.audio) { %>
+    <span class="audio-control playing">Pause Audio</span>
+    <audio autoplay>
+        <% _.each(zone.audio.files, function(file) { %>
```

Comment:
What would happen here if there were multiple files? Or is this what you do to load the same audio but with different encoding?